contents/blog/aruba-hackathon.md

```diff
@@ -0,0 +1,185 @@
+
+- **Team DataBeach:** Marius Andra, Frank Hamand and Harry Waye
+
+Dubbed DataBeach because Frank and Harry started building the feature while sipping Piña Coladas by the beach, DataBeach is all about our long-term vision of  [simplifying the modern data stack](/blog/modern-data-stack-sucks) for startups. Less time spent wrangling data = more time shipping products. 
+
+![databeach](../images/blog/aruba/data-beach.png)
+
+The MVP they built consists of custom tables that are created and queried through the PostHog UI and API. These tables provide a way to store and query data from sources like Stripe, Hubspot (see above), Intercom, and more, along with data from PostHog.
+
+There's more to build before PostHog is ready to be your data warehouse, but we're working on it. Keep an eye on our [public roadmap](/roadmap) for updates.
+
+## Building a Zendesk killer in PostHog
+
+- **Team Arubug:** Tiina Turban, Simon Fisher, Paul D'Ambra, Cameron DeLeone, and Cory Watilo
+
+Anecdotally, our direct-to-an-engineer support is a big reason why people love us. But we've run into a few of inefficiencies as we've grown, such as:
+
+- Support requests that lack the context an engineer needs to fix it
+- Lack of trends and aggregated info about support requests
+- Difficulty managing multiple sources for support requests
+
+Enter team Arubug, who decided there had to be a better way.
+
+// TODO: Video of site app
+
+The team started by [building a site app](/tutorials/build-site-app) for bug reports based on our existing [Feedback Widget](/apps/feedback-widget), which sends a `$bug_report` event to PostHog.
+
+// TODO: Screenshot of dashboard
+
+These reports feed into a dashboard that tracks bug reports, helping us to identity trends. Bugs can broken down as tables with relevant properties, and session replays, attached.
+
+// TODO: Screenshot of bug report table 
+
+The team also built a communication tab into bug reports, so support can send emails and leave notes on tickets with additional context without leaving the app. Every email is a ClickHouse event tied to the initial UUID of the report event, with emails (for now) sent and received via Zapier.
+
+// TODO: Screenshot of communication tab
+
+After the hackathon we've reviewed the plan:
+
+- We realised that there's another problem internally. Due to the increased complexity of our product, it’s become harder for a single engineer to provide good support for all of our products. Support heroes also shield product teams from feedback about their product, which means it’s harder to iterate on a product. So we're going to use this in-app integration to split out to the more specific team helping us do triaging effectively.
+
+- We've decided at this point not to continue the work on the communication tab and instead integrate better with Zendesk. That said, the way we handle support can be used by customers too – there will be a blog post coming to explain this further.
```

Comment:
```suggestion
After the hackathon, we reviewed the plan and realised:

1. Due to the increased complexity of our product, it’s become harder for a single engineer to provide good support for all of our products.

2. Support heroes shield product teams from important feedback about their product, blocking an important feedback loop.

Consequently, we decided to use this new in-app integration to triage support requests by product/team, and cease work on the communication tab and focus on deeper integration with Zendesk.

// TODO: Screenshot of bug request help thingy

Now, we're routing all support requests through the Help menu in the PostHog, making it easier for us to triage tickets and gather the context (including session replays) to solve your problems efficiently. 
```

Reply:
@tiina303 @pauldambra Suggested tweaks to the above. Let me know if this is good to go!

Reply:
Maybe just "and keep using the integration with Zendesk."